templates/components/products/price.html

```diff
@@ -2,19 +2,23 @@
     {{> components/products/price-range price=price schema_org=schema_org}}
 {{else}}
     {{#if price.with_tax}}
-        <div class="price-section price-section--withTax rrp-price--withTax">
-            {{#if price.rrp_with_tax}}
-                {{lang 'products.retail_price'}}
-                <span data-product-rrp-with-tax class="price price--rrp"> {{price.rrp_with_tax.formatted}}</span>
-            {{/if}}
+        <div class="price-section price-section--withTax rrp-price--withTax" {{#unless price.rrp_with_tax}}style="display: none;"{{/unless}}>
+            {{lang 'products.retail_price'}}
+            <span data-product-rrp-with-tax class="price price--rrp">
+                {{#if price.rrp_with_tax}}
```

Comment:
same here with moving the if statement up a bit? maybe you had a reason for showing the text though.

Reply:
Ah, actually, if I understand this right, you can probably take these if statements out entirely-- since you're basically changing the way you chose wether or not to display with the `style="display: none;"` That should probably do it.

Reply:
I think I need the IF statement because I am not sure if `price.rrp_with_tax` exists....not sure how stencil acts if I try to print a value that is not there.  If if does not break anything to try and print a value that does not exist (ex:`{{price.rrp_with_tax.formatted}}`) then i can leave out those IFs.

The style display none ensures that the "Retail price" label will not be seen until toggled on by the JS.

Reply:
I just mean that you're already checking if theres a value for price with tax above:
```
<div class="price-section price-section--withTax rrp-price--withTax" {{#unless price.rrp_with_tax}}style="display: none;"{{/unless}}>
```

you dont probably need it again 
```
  {{#if price.rrp_with_tax}}
```
right?

Reply:
I'll test what happens if I exclude it....the `{{#unless..}}` conditional will make the element hide when there is not a retail price value to enter....im just not sure how stencil will react if i try to access a non-existent property